.pre-commit-config.yaml

```diff
@@ -39,6 +39,12 @@ repos:
       pass_filenames: false
       entry: python -m utils.check_backend_completeness
       language: python
+    # - id: generate-backend-completeness
+    #   name: generate-backend-completeness
+    #   pass_filenames: false
+    #   entry: python -m utils.generate_backend_completeness
+    #   language: python
+    #   additional_dependencies: [polars, jinja2]
```

Comment:
is this meant to be uncommented?

Reply:
Sorry, I didn't come back to this yet!

Thing is: this would run and generate the expected file, however since _some file_ changed, overall pre-commit would fail. Run it again, and it would be all good since it would generate the same exact file.

It's also the first time I use a custom script in pre-commit, so maybe there is some configuration to tweak, I am just not aware of.

If you have any input on this, otherwise I can come back later today :)

Reply:
> Thing is: this would run and generate the expected file, however since some file changed, overall pre-commit would fail. Run it again, and it would be all good since it would generate the same exact file.

this sounds like expected behaviour, no? this is what I'd want: if it modifies a file, exit 1, and then you need to re-stage and re-commit, else exit 0